File: ios/CustomMethod.m
//
//  CustomMethod.m
//  Bridging
//
//  Created by Nirmalsinh Rathod on 29/02/2024.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>


@interface RCT_EXTERN_MODULE(CustomMethod, NSObject)
  
RCT_EXTERN_METHOD(CallMethod : (NSString *) param)

@end

File: ios/CustomMethod.swift
//
//  CustomMethod.swift
//  Bridging
//
//  Created by Nirmalsinh Rathod on 29/02/2024.
//

import Foundation

@objc(CustomMethod) class CustomMethod: NSObject {
  @objc static func requiredMainQueue()-> Bool {return true}
  
  @objc public func CallMethod (_ param :String) {
    print(param)
    RNEventEmitter.emitter.sendEvent(withName: "onReady", body: [param])
  }
}
